OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -60,6 +60,7 @@ MetaModelEditor::MetaModelEditor(QWidget *pParent)
   : BaseEditor(pParent), mLastValidText(""), mTextChanged(false), mForceSetPlainText(false)
 {
   mXmlDocument = XMLDocument(this);
+  connect(mpPlainTextEdit,SIGNAL(textChanged()),this,SLOT(updateAllOrientations()));
```

Comment:
I just realized this does not work anyway, since the actual data is not updated from the text until user switches to diagram view. I will remove the line. 

Right now the view is not updated when user saves the model, only when switching to diagram view. I could not figure out how to make an update while saving.